tokio/CHANGELOG.md

```diff
@@ -1,3 +1,58 @@
+# 1.30.0 (August 6, 2023)
+
+This release bumps the MSRV of Tokio to 1.63. ([#5887])
+
+### Changed
+
+- tokio: reduce LLVM code generation ([#5859])
+- io: support `--cfg mio_unsupported_force_poll_poll` flag ([#5881])
+- sync: make `const_new` methods always available ([#5885])
+- sync: avoid false sharing in mpsc channel ([#5829])
+- rt: pop at least one task from inject queue ([#5908])
+
+### Added
+
+- sync: add `broadcast::Sender::new` ([#5824])
+- net: implement `UCred` for espidf ([#5868])
+- fs: add `File::options()` ([#5869])
+- time: implement extra reset variants for `Interval` ([#5878])
+
+### Removed
+
+- tokio: removed unused `tokio_*` cfgs ([#5890])
+- remove build script to speed up compilation ([#5887])
+
+### Documented
+
+- sync: mention lagging in docs for `broadcast::send` ([#5820])
+- runtime: expand on sharing runtime docs ([#5858])
+- io: use vec in example for `AsyncReadExt::read_exact` ([#5863])
+- time: mark `Sleep` as `!Unpin` in docs ([#5916])
+- process: fix `raw_arg` not showing up in docs ([#5865])
+
+### Unstable
+- rt: add runtime ID ([#5864])
+- rt: initial implementation of new threaded runtime ([#5823])
+
+[#5887]: https://github.com/tokio-rs/tokio/pull/5887
+[#5859]: https://github.com/tokio-rs/tokio/pull/5859
+[#5859]: https://github.com/tokio-rs/tokio/pull/5881
```

Comment:
This link appears twice. Please double check the links. We usually sort them to make them easier to see.

Reply:
@Darksonn done, I did originally consider sorting but I looked at a few prior entries and they weren't sorted but I agree it is best

Reply:
I guess it depends on who wrote it. I always sort them.